data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,138 @@
-
+{
+    "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.1.0/virtualcollections/virtualcollections.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$comment": "DigitalObject Version 0.1.0",
+    "type": "object",
+    "properties": {
+        "ods:id": {
+            "type": "string",
+            "description": "PID of the object.",
+            "pattern": "^https:\/\/hdl\\.handle\\.net\/20\\.5000\\.1025\/.{3}-.{3}-.{3}$",
+            "examples": [
+                "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"
+            ]
+        },
+        "rdf:type": {
+            "type": "string",
+            "description": "The type is always VirtualCollections. https://www.w3.org/TR/rdf12-schema/#ch_type",
+            "const": "VirtualCollections"
+        },
+        "ods:version": {
+            "type": "integer",
+            "description": "The version of the object.",
+            "examples": [
+                1
+            ]
+        },
+        "dcterms:creator": {
```

Comment:
In the annotation model we have two fields as required:
```
      "required": [
        "ods:id",
        "ods:type"
      ]
```

Reply:
added